tests/test-class-amp-wp-styles.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * Tests for AMP_WP_Styles.
+ *
+ * @package AMP
+ * @since 0.6
+ */
+
+/**
+ * Tests for AMP_WP_Styles.
+ *
+ * @covers AMP_WP_Styles
+ */
+class Test_AMP_WP_Styles extends WP_UnitTestCase {
+
+	/**
+	 * Tear down.
+	 */
+	public function tearDown() {
+		global $wp_styles;
+		parent::tearDown();
+		$wp_styles = null;
+	}
+
+	/**
+	 * Get wp_styles().
+	 *
+	 * @return WP_Styles|AMP_WP_Styles Styles.
+	 */
+	protected function amp_wp_styles() {
```

Comment:
This method can be removed and `AMP_Theme_Support::override_wp_styles()` can be used instead if/when the comment left on the `override_wp_styles()` are applied.